CHANGELOG.md

```diff
@@ -11,3 +11,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add models to handle rooms, meetings, labels and groups
 
 ## [Unrealeased]
+
+### [Added]
```

Comment:
My understanding was that you wanted to use reactQuery. Is it still the plan?

Reply:
reactQuery will be another layer to manage the result to api calls, when this is needed. It will be used later